pom.xml

```diff
@@ -29,7 +29,7 @@
 
     <groupId>com.github.cafapi.util.flywayinstaller</groupId>
     <artifactId>util-flywayinstaller-aggregator</artifactId>
-    <version>1.1.0-SNAPSHOT</version>
+    <version>1.0.1-SNAPSHOT</version>
```

Comment:
Hi Vishal,
you're going backwards in version numbering here. Look at US 551015 for directions on how to update version numbers.
Looking at the history I do not see recent changes, can this not just be released?

Reply:
@frankmccarry I think 1.0.1 is the correct version to move to for this release, the update of the psql driver would not constitute a new feature so the next patch version from the last release looks correct to me.